tests/tape/tapes/test_cv_param_shift.py

```diff
@@ -353,7 +352,7 @@ def test_no_poly_xp_support_variance(self, mocker, monkeypatch, caplog):
 
 
 class TestExpectationQuantumGradients:
-    """Tests for the quantum gradients of various gates
+    """Tests for the quantum gradients of qml.various gates

```

Comment:
:smile: search & replace 
```suggestion
    """Tests for the quantum gradients of various gates

```